src/libs/ReportUtils.ts

```diff
@@ -2586,7 +2579,7 @@ function getMoneyRequestReportName(report: OnyxEntry<Report>, policy?: OnyxEntry
     const reportFields = isReportSettled ? report?.fieldList : getReportFieldsByPolicyID(report?.policyID ?? '-1');
     const titleReportField = getFormulaTypeReportField(reportFields ?? {});
 
-    if (titleReportField && report?.reportName && reportFieldsEnabled(report)) {
+    if (titleReportField && report?.reportName && isPaidGroupPolicyExpenseReport(report)) {
```

Comment:
These checks need to be updated from isPaidGroupPolicyExpenseReport to isControlPolicyExpenseReport

Report fields are going to be a control only feature (unless it's the title field)

Reply:
@thienlnam hi, this logic seems to actually check for the title field and return the report name according to formula. So we probably do want this as it is right now? 

Same for [this line](https://github.com/Expensify/App/pull/44695/files#diff-577fcc5f3a5e4930916ff310e61c38edfa093792fe3ab55b9a30d2aecd8811dbR3991) which is creating an optimistic report and getting the title field.

The check on [this line](https://github.com/Expensify/App/pull/44695/files#diff-37a97d708344580593d4b3d79658ef31ae2410b49700e44ef0c58662c2a53f98R65) and is for showing the Report Fields in the `MoneyReportView`. Changing this to `isControlPolicyExpenseReport` means we would hide the title field, since the render logic is coupled with report fields. I could fix it if you suggest (We would change the [`getAvailableReportFields`](https://github.com/Expensify/App/blob/58e257532290aa0228f1c668ed461a15f1b2b136/src/libs/ReportUtils.ts#L2564-L2595) to two functions, one which returns the title, other handling remaining fields)  

If we change the check, it might flag as a regression since Report Fields are still configurable on OldDot for Collect Policies:

<Details><Summary>Video</Summary>
 

https://github.com/user-attachments/assets/684bea1f-64b5-40cb-a42d-1cc189331ee8

</Details>

Reply:
Ah yeah that's a fair point about how they are coupled right now. In that case, we probably don't need to do anything right now - we'll handle that later when we remove it from collect polcies